### PR TITLE
Remove click versions from everywhere except `setup.py`

### PR DESCRIPTION
**Pull request by rahul-tuli**
This PR removes `click` versions from everywhere except `setup.py` to have one source of truth where `click` `version` is controlled

### CODE REVIEW

File: examples/aws-sagemaker/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 rich==12.3.0
 boto3==1.22.5
-click<8.1
\ No newline at end of file
+click
```
Comment (bfineran) [inline examples/aws-sagemaker/requirements.txt:3]: why not delete?

Comment (rahul-tuli) [reply examples/aws-sagemaker/requirements.txt:3]:
```suggestion
```

**Comment by markurtz** [comment]
@rahul-tuli let's get all references deleted now given that all installs should require it now

**Review by bfineran — COMMENTED** [review]
actually, thinking about tutorials like `aws-sagemaker`, deepsparse is not actually installed by the user (installed within a container in the tooling) so `click` won't be available to run the examples

File: examples/aws-sagemaker/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 rich==12.3.0
 boto3==1.22.5
-click<8.1
\ No newline at end of file
+click>=8.0
```
[the remark this answers is not in the document]
Comment (bfineran) [reply examples/aws-sagemaker/requirements.txt:3]: net net, click is required for this example and not installed via deepsparse. if a user were to install deepsparse as well, the pinned version would take priority. when click is upgraded in deepsparse we shouldn't need to update this line later

**Review by bfineran — APPROVED** [review]

**Review by corey-nm — DISMISSED** [review]

**Review by bfineran — APPROVED** [review]

**Review by KSGulin — APPROVED** [review]

**Review by corey-nm — APPROVED** [review]

**Pull request merged**